res/skins/LateNight/waveform.xml

```diff
@@ -46,6 +46,7 @@
               <Align>bottom|right</Align>
               <Color>#FF0000</Color>
               <TextColor>#FFFFFF</TextColor>
+              <EndIcon>skins:LateNight/palemoon/buttons/btn__beatjump_right.svg</EndIcon>
```

Comment:
This is quite nasty, but I couldn't think of an elegant way to customise the the end marker only. Any suggestion to do that better without needing a major refactor of `Mark`?